packages/redux/MIGRATION.md

```diff
@@ -0,0 +1,2166 @@
+This guide will help you migrate `@farfetch/blackout-redux` to the latest version.
+
+# Table of Contents
+
+- [Migrating from @farfetch/blackout-core](#migrating-from-@farfetch/blackout-core)
+
+  - [Addresses](#addresses)
+  - [Analytics](#analytics)
+  - [Authentication](#authentication)
+  - [Bags](#bags)
+  - [Brands](#brands)
+  - [Categories](#categories)
+  - [Checkout](#checkout)
+  - [Contents](#contents)
+  - [Designers](#designers)
+  - [Entities](#entities)
+  - [Forms](#forms)
+  - [Helpers](#helpers)
+  - [Locale](#locale)
+  - [Loyalty](#loyalty)
+  - [Management](#management)
+  - [Merchants Locations](#merchants-locations)
+  - [Orders](#orders)
+  - [Payments](#payments)
+  - [Products/Details](#products/details)
```

Comment:
```suggestion
  - [Products/Details](#productsdetails)
```

Reply:
I used VSCode's to test the links and they worked there. Fixed, thanks!